service_container/service_subscribers_locators.rst

```diff
@@ -457,6 +457,8 @@ will share identical locators among all the services referencing them::
             // ...
             'logger' => new Reference('logger'),
         ];
+        
+        $myService = $container->getDefinition(MyService::class);
```

Comment:
```suggestion
        $myService = $container->findDefinition(MyService::class);
```

Both work, but `findDefinition` is a little more robust, as it would find `MyService` even if it were an alias to another service.